pkg/virt-handler/device-manager/pci_device_test.go

```diff
@@ -95,6 +97,12 @@ pciHostDevices:
 	})
 
 	It("Should validate DPI devices", func() {
+		// Allow the regular functions to be called for all the other devices, they're harmless.
+		// Just force the driver to NOT vfio-pci to ensure they all get ignored.
+		mockPCI.EXPECT().GetDeviceIOMMUGroup(pciBasePath, gomock.Any()).AnyTimes()
+		mockPCI.EXPECT().GetDeviceDriver(pciBasePath, gomock.Any()).Return("definitely-not-vfio-pci", nil).AnyTimes()
+		mockPCI.EXPECT().GetDeviceNumaNode(pciBasePath, gomock.Any()).AnyTimes()
+		mockPCI.EXPECT().GetDevicePCIID(pciBasePath, gomock.Any()).AnyTimes()
```

Comment:
Can you please move this code duplication in a `BeforeEach`?

Reply:
With this code in the `BeforeEach` I couldn't make the new test work. I guess that for the `MockDeviceHandler` we can't force functions for specific PCI address after we allowed functions to be called for any other device.